src/python/grpcio/grpc/_channel.py

```diff
@@ -1484,30 +1643,37 @@ def __init__(self, target, options, credentials, compression):
         if cygrpc.g_gevent_activated:
             cygrpc.gevent_increment_channel_count()
 
-    def _process_python_options(self, python_options):
+    def _process_python_options(
+            self, python_options: Optional[ChannelArgumentType]) -> None:
```

Comment:
Why `Optional`?

Reply:
Same as `core_options`, it can be an empty list but not `None`, will remove `Optional` here.